vtune/README.md

```diff
@@ -0,0 +1,81 @@
+## VTune analysis
+
+First, the code has been run under VTune on DevCloud "cluster" and we obtained the following picture:  
+
+![](./img/vtune/old/new_core.c_ua1.png) 
+
+As we can see, VTune marks the most CPU consuming task as "Outside any known module", and lots of lines are grayed out and marked "unreliable" when pointed with a mouse. It looks like we do not have proper access to hardware counters on Intel machine, so we went to a local laptop to see more detailed results. Thus the following picture has been obtained:
```

Comment:
So, what do you expect? Please, refer to Advisor for this task to justify that we expect a function here.